src/lib/containers/oauth/OAuthManagement.tsx

```diff
@@ -87,6 +99,39 @@ export const OAuthManagement: React.FunctionComponent = () => {
           isShowingConfirmModal={isShowingConfirmModal}
         />
       )}
+      <Modal
+        show={isShowingVerification}
+        animation={false}
+        backdrop="static"
+        onHide={() => setIsShowingVerification(false)}
+      >
+        <Modal.Header closeButton>
+          <Modal.Title>
+            <Typography variant="headline1">Submit Verification</Typography>
+          </Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <p>
+            In order to verify an OAuth client please send an email to{' '}
+            <a href={'mailto:synapseinfo@sagebase.org'}>
+              synapseinfo@sagebase.org
+            </a>
+            .
+          </p>
+          <b>Please list the following items in your email:</b>
```

Comment:
Likewise, please update this copy to:
Please list the following items in your request: